contributing/bazel_kokoro.md

```diff
@@ -117,22 +117,19 @@ If your component depends on non-source resources, you can add a bundle target t
 with the following additions:
 
 ```
+load("//tools/build_defs/apple:resources.bzl", "apple_bundle_import")
```

Comment:
I think we don't want this change either

Reply:
Haha, yes. I just removed it. couldn't find it in a specific commit that I can revert easily. Thanks for pointing out.